BrainPortal/app/views/groups/show.html.erb

```diff
@@ -110,28 +113,65 @@
 
   <%
     group_members  = @group.users
+    group_editors  = @group.editors || []
+    group_editors << @group.creator unless group_editors.include? @group.creator
+    group_viewers  = group_members - group_editors
     open_invites   = Invitation.where(sender_id: current_user.id, group_id: @group.id, active: true).to_a
-  %>
 
-  <%= show_table(@group, :as => :group, :header => 'Members', :edit_condition => @group.can_be_edited_by?(current_user) && (!current_user.has_role?(:normal_user) || group_members.count > 1 || open_invites.count > 0 )) do |t| %>
+    if @group.is_a?(WorkGroup) then # for now editors are only allowed on WorkGroups
```

Comment:
Please redesign this layout, it's super ugly.

I suggest you list the members only once, with an asterisk beside them when they are editor (and a proper note about this).